Makefile

```diff
@@ -38,6 +38,11 @@ test:
 
 	coverage3 report -m
 	[ -f "/usr/bin/coveralls" ] && [ -n "$(COVERALLS_REPO_TOKEN)" ] && coveralls || echo
+	
+	set -ex
+	./tests/cli/test_blueprints_sanity.sh
+	grep RESULT_STRING /var/tmp/beakerlib-*/TestResults | grep -v PASS && exit 1
```

Comment:
> This condition is the wrong way around (PASSing tests are marked as FAIL).

I believe this is because of how make executes this pipeline. I'll move into a bash script together with a common setup and see how it goes.